<launch>
 
   <group ns="imu_pub">
   <node pkg="sar_localization" name="imu" type="serial_message"/>
   </group>
 
   <group ns="csi_pub">
   <node pkg="sar_localization" name="csi" type="log_csi_stream"/>
   </group>

	 <group ns="sar">
	 <node pkg="sar_localization" name="sar" type="listener"/>
	 </group>
 
 
</launch>
